### DISCUSSION TITLE
[FEATURE] Proper docs for DataTable conditional formatting

### DISCUSSION THREAD

**Discussion by thomktz** [bokeh/bokeh#13955] [Documentation]
### Problem description

I want to add proper examples/docs/blog post to showcase the Datatable conditional formatting feature from this issue/PR: #13682

### Feature description

Unsure - where is the best place to put examples etc?

### Potential alternatives

/

**Comment by bryevdv** [comment]
Hi @thomktz more docs examples would certainly be welcome. 

* Our blog posts at [blog.bokeh.org](https://blog.bokeh.org/) are actually hosted on [Medium](https://medium.com/), so the way to contribute a blog post would be to create your own account on Medium, and then author the post you want to be included in the Bokeh blog. We can then publish it to the the blog under your name. 

* Examples are just scripts added under the [`examples` directory](https://github.com/bokeh/bokeh/tree/branch-3.5/examples). You can just make a PR to add the example script you want to see added. Please just have a look at some existing example scripts and try to match their "complexity level" and style.

  Most of the subdirectories there correspond directly to sections of the users guide (e.g. [`examples/interaction/tools`](https://github.com/bokeh/bokeh/tree/branch-3.5/examples/interaction/tools) corresponds to https://docs.bokeh.org/en/latest/docs/user_guide/interaction/tools.html These directories also determine the structure of the gallery. 

   I mention this just because you'll want to give some consideration to where the best place to add a new example is. If there is an example in a good place, that makes it much easier to add a gallery entry or new users guide section in the future. 

* Narrative user guide docs are [Sphinx](https://www.sphinx-doc.org/en/master/) docs under [`docs/bokeh`](https://github.com/bokeh/bokeh/tree/branch-3.5/docs/bokeh). If you have an example already added, you can edit the `.rst` sources for the chapter to you want to add to and use the `bokeh-plot` directive to include the example inline in the docs you write. There is also a [`gallery.json`](https://github.com/bokeh/bokeh/blob/branch-3.5/docs/bokeh/source/docs/gallery.json) that is where you would add the example to the gallery (will also need two thumbnails to be added). 

I'd say the blog post is something you can do independently any time you like. Just point us as the article you want to see posted under our blog site. 

Otherwise, I'd suggest starting by adding an example file or two, then later it can be added to the gallery or included in new user guide content.
